### PR TITLE
Add 8-player, MIT-licensed, standalone game for multiplayer testing

### PR DESCRIPTION
**Pull request by garbear**
This adds Mr.Boom (https://github.com/libretro/mrboom-libretro) for testing the new multiplayer and button-mapping framework added in https://github.com/xbmc/xbmc/pull/13499.

Previously, we just included 2048, which was added so skinners could test support for the In-Game GUI project last summer. The Controller Topology project is the next big addition for RetroPlayer, because for the first time we can model how controllers connect to and map to each other for all of gaming history. Mr.Boom is sufficient for testing because it models both [mapping](https://github.com/kodi-game/game.libretro.mrboom/blob/master/game.libretro.mrboom/resources/buttonmap.xml) and [connecting](https://github.com/kodi-game/game.libretro.mrboom/blob/master/game.libretro.mrboom/resources/topology.xml) multiple controllers and doesn't use ROMs, so it'd be nice to expand testing for this project.

### CODE REVIEW

File: game.libretro.mrboom/game.libretro.mrboom.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+game.libretro.mrboom https://github.com/kodi-game/game.libretro.mrboom.git master
```
Comment (Rechi) [inline game.libretro.mrboom/game.libretro.mrboom.txt:1]: `https://github.com/kodi-game/game.libretro.mrboom`

Comment (garbear) [reply game.libretro.mrboom/game.libretro.mrboom.txt:1]: oops. I copied the URL from github and it thought the .git was a good idea. Should I fix?

Comment (Rechi) [reply game.libretro.mrboom/game.libretro.mrboom.txt:1]: yes, it doesn't build otherwise

Comment (MartijnKaijser) [reply game.libretro.mrboom/game.libretro.mrboom.txt:1]: already fixed

**Pull request merged**